docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -192,10 +224,14 @@ public static void Example()
                     }
                 }
 
+                // Get the buffer of bytes
+                int imgSize = LoadDataIntoBuffer(file, ref imgData);
```

Comment:
>imgSize [](start = 20, length = 7)

what is imgSize used for?

Reply:
The new code has changes for this. I throw an exception if imgSize is less than zero. This part has been moved to ImageLoader.cs.